# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/ipc/BrowserParent.h
# Commit: ced775b2651e
# Full Hash: ced775b2651e581ecf7d4ade35e39f1791f703d0
# Author: Henri Sivonen <hsivonen@hsivonen.fi>
# Date: 2019-05-07 14:47:21
# Description:
#   Bug 1544632 - Check for BrowserParent destruction before sending StopIMEStateManagement IPC message. r=masayuki
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D30007
# ==============================================================================

diff -r 78ccad22a01c -r ced775b2651e dom/ipc/BrowserParent.h
--- a/dom/ipc/BrowserParent.h	Mon May 06 11:49:05 2019 +0000
+++ b/dom/ipc/BrowserParent.h	Mon May 06 12:00:03 2019 +0000
@@ -573,6 +573,8 @@
   // PuppetWidget's origin to absolute screen coordinates in the child.
   LayoutDeviceIntPoint GetClientOffset();
 
+  void StopIMEStateManagement();
+
   /**
    * Native widget remoting protocol for use with windowed plugins with e10s.
    */
